components/docs/docs.html

```diff
@@ -65,6 +65,9 @@ <h1 class="component-heading">Components</h1>
           <li class="menu-item">
             <a name="chips" class="component-btn  a-no-underline" href="#">Navigation</a>
```

Comment:
when attributes are long can have them on multiple line for better readabilty.

Reply:
understood